### ISSUE TITLE
Publishers page is missing pagination

### ISSUE THREAD

**Issue by tkeuneman** [TerriaJS/magda#1010]
### Problem description
You could scroll to infinity and beyond. Bring back the pagination

### Problem reproduction steps
http://magda-dev.terria.io/publishers

**Comment by AlexGilleran** [comment]
@tkeuneman we did this deliberately to avoid having to build search functionality for publishers - are you sure you want to revert it?

**Comment by tkeuneman** [comment]
@AlexGilleran Can't we have pagination without search?

**Comment by tkeuneman** [comment]
I'm also adding a UX task to look at search for publishers.

**Comment by t83714** [comment]
One feasible option could be pagination at the frontend.
i.e. 
- still load all publisher records into memory on page open
- paginate it in memory by array Index
- we may be able to do the search in memory as well
The only drawback with this solution is it is only feasible if total no. of publisher records is not too big.

**Comment by AlexGilleran** [comment]
> @AlexGilleran Can't we have pagination without search?

Yeah but it means you've got to guess which page the publisher you want is on, instead of using ctrl-f 🤷‍♂️
